CMakeLists.txt

```diff
@@ -22,6 +22,10 @@ endif()
 
 find_package(ROOT REQUIRED CONFIG)
 
+if(NOT DEFINED PYTHON_EXECUTABLE)
+  find_package(PythonInterp ${ROOT_PYTHON_VERSION} REQUIRED QUIET)
```

Comment:
I see that we do `set(ROOT_PYTHON_VERSION @PYTHON_VERSION_STRING_Development_Main@)` in `ROOTConfig.cmake.in`, so this is fine. As long as this tries to find the main/highest Python version that PyROOT has been built with, it's ok.